.github/actions/tf-deploy/action.yml

```diff
@@ -45,5 +52,6 @@ runs:
         ARM_CLIENT_SECRET: ${{ inputs.terraform_arm_client_secret }}
         ARM_SUBSCRIPTION_ID: ${{ inputs.terraform_arm_subscription_id }}
         ARM_TENANT_ID: ${{ inputs.terraform_arm_tenant_id }}
+        OCTOPUS_VERSION: ${{ env.VERSION }}
```

Comment:
Can we inherit these variables (i.e. `ARM_CLIENT_SECRET`, `ARM_SUBSCRIPTION_ID`, `OCTOPUS_VERSION`, etc.) to the other jobs?  Just wondering because this may assist with refactoring for the terraform plan and terraform apply.

Reply:
Can you clarify what you're looking for? Those variables are already present in the jobs you cited, and I'm not aware of any refactor that's needed at this time.

Reply:
Ah yes, was thinking of using the `needs` dependency to reuse the variables.  However, I do not think it will work here.